### ISSUE TITLE
homepage - tab order

### ISSUE THREAD

**Issue by IdaAkiwumi** [religious-ecologies/omeka-s-theme#20]
Tab order for the image links and the table seems out of order. After tabbing from Schedules image/link to SCHEDULES PER COUNTY table links, the tab actually skips that table and goes to the table on the far right SCHEDULES PER DENOMINATION (Southern Baptist Convention | 13146 specifically), before it goes to the bottom table.

Visually, this lets the page jump up (when the user should be tabbing down), then to the right going down, then moves back to the left.

**Comment by kimisgold** [comment]
In https://github.com/religious-ecologies/omeka-s-theme/commit/4a386e456d21e0660594d3831c1bece5e1baf5ec, I pulled out the jquery plugin that was causing this issue. The homepage doesn't have the same variable height tiling, but I think it creates a more logical tab order. @IdaAkiwumi, can you take a look and see if this resolves the issue?
